sql/mysql/sqlspec.go

```diff
@@ -34,6 +38,26 @@ func UnmarshalSpec(data []byte, unmarshaler schemaspec.Unmarshaler, v interface{
 	return fmt.Errorf("mysql: failed unmarshaling spec. %T is not supported", v)
 }
 
+// MarshalSpec marshals v into an Atlas DDL document using a schemaspec.Marshaler.
```

Comment:
is v a known name for interface?

Reply:
https://cs.opensource.google/go/go/+/refs/tags/go1.17.2:src/encoding/json/encode.go;l=158